features/step_definitions/operators.rb

```diff
@@ -160,3 +160,47 @@
   cb_name = 'operator_channel_name' unless cb_name
   cb[cb_name] = cluster_version('version').channel.split('-')[1]
 end
+
+Given /^operator #{QUOTED} becomes "([^"]*)" within(?: (\d+) seconds)?$/ do | operator_name, conditions, timeout |
+  ensure_admin_tagged
+
+  expected = {}
+  reg = /(\_|available|non-available)\/(\_|progressing|non-progressing)\/(\_|degraded|non-degraded)/
```

Comment:
I don't see need to use underscores `_` and quotes. When you get the string we could `conditions.split("/")`, then check if we have `non-` prefix for each word and then check which word do we have. I didn't have time to write implementation od this today but if you don't have time for it, I can hopefully do it tomorrow.

Reply:
I have no time to work on this currently, will give it a try in another time.

Reply:
I changed the code to get conditions by `conditions.split("/")`, then check `non-` prefix to get the word we want.

To call the step, see below,
```
    Given operator "kube-apiserver" becomes "progressing" within 100 seconds
    Given operator "kube-apiserver" becomes "available/non-progressing/non-degraded" within 1200 seconds
    Given operator "kube-apiserver" becomes "available" within 100 seconds
    Given operator "kube-apiserver" becomes "" within 100 seconds

```

Reply:
@akostadinov, Here is test results: Runner-v3-smoke/2709/console, when you are available, please help to review, thanks.